R-package/tests/testthat/test_Predictor.R

```diff
@@ -442,8 +436,52 @@ test_that("predictions for multiclass classification are returned as matrix", {
     expect_true(is.matrix(pred))
     expect_equal(nrow(pred), nrow(X))
     expect_equal(ncol(pred), 3L)
-    pred <- predict(model, X, rawscore = TRUE)
+    pred <- predict(model, X, type = "raw")
     expect_true(is.matrix(pred))
     expect_equal(nrow(pred), nrow(X))
     expect_equal(ncol(pred), 3L)
 })
+
+test_that("predict type='response' returns integers for classification objectives", {
```

Comment:
```suggestion
test_that("predict type='response' returns predicted class for classification objectives", {
```

The significant thing here is that the result is the predicted classes, not just that they're integers. Would you please consider this rewording?